src/pages/Editor.vue

```diff
@@ -1386,6 +1388,9 @@ export default {
     },
   },
   methods: {
+    forceUpdate() {
```

Comment:
just `reRender() {` is fine no?

Reply:
reRender function name? yess,will change that